server/routes/browser.py

```diff
@@ -26,4 +28,12 @@ def kg_main():
 
 @bp.route('/<path:dcid>')
 def kg_entity(dcid):
```

Comment:
can update the function names "kg_main" and "kg_entity" as well?

Reply:
done